pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -71,6 +72,7 @@ func NewCmdAddRoleToGroup(name, fullName string, f *clientcmd.Factory, out io.Wr
 		},
 	}
 
+	cmd.Flags().StringVarP(&options.RoleBindingName, "rolebinding-name", "N", "", "name of the rolebinding to modify or create")
```

Comment:
We have a few cases of this, such as in [new-app](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/newapp.go#L229), [rsh](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/rsh.go#L99), and [new-build](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/newbuild.go#L126)